content/en/getting_started/agent/_index.md

```diff
@@ -120,7 +120,7 @@ For help troubleshooting the Agent:
 [8]: https://github.com/DataDog/datadog-agent/tree/master/Dockerfiles/agent
 [9]: /agent/guide/agent-configuration-files/#agent-main-configuration-file
 [10]: https://app.datadoghq.com/account/settings#api
-[11]: /getting_started/agent/configuration/
+[11]: /getting_started/agent/#configuration
```

Comment:
This is just linking to this file, this section. Is there a guide for Agent config?

Reply:
Updated the text to match the link, so we're good to go 👍